iib/workers/tasks/build.py

```diff
@@ -418,6 +418,22 @@ def _verify_index_image(resolved_prebuild_from_index, unresolved_from_index):
         )
 
 
+def _verify_labels(bundles):
+    """
+    Verify that the required labels are set on the input bundles.
+
+    :param list bundles: a list of strings representing the pull specifications of the bundles to
+        add to the index image being built.
+    :raises iib.exceptions.IIBError: if one of the bundles does not have the correct label value.
+    """
+    conf = get_worker_config()
+    for bundle in bundles:
+        labels = get_image_labels(bundle)
```

Comment:
I think we are making more and more requests to the `get_image_labels` method which in-turn keeps calling skopeo inspect. Do you think it makes sense to store the labels for a bundle in the database?

Reply:
I agree it's a bit wasteful, but storing it in the database is more wasteful since the worker would have to make several API requests (patch and get) and we'd additionally have to store this data when we don't need to. If you'd like we could consider a cache such as the memory backend of dogpile cache:
https://dogpilecache.sqlalchemy.org/en/latest/api.html#memory-backends

I'd like to avoid doing that in this PR though.